scripts/download/lima.mjs

```diff
@@ -10,7 +10,7 @@ const limaRepo = 'https://github.com/rancher-sandbox/lima-and-qemu';
 const limaTag = 'v1.17';
 
 const alpineLimaRepo = 'https://github.com/lima-vm/alpine-lima';
-const alpineLimaTag = 'v0.2.5';
+const alpineLimaTag = 'v0.2.6';
```

Comment:
> `Error: Error downloading https://github.com/lima-vm/alpine-lima/releases/download/v0.2.6/alpine-lima-rd-3.14.3-x86_64.iso.sha512sum`

I think we forgot to actually ship 0.2.6? https://github.com/lima-vm/alpine-lima/releases/tag/v0.2.6 doesn't have any assets (other than the source tarballs).